gems/aws-sdk-core/lib/aws-sdk-core/instance_profile_credentials.rb

```diff
@@ -257,7 +258,7 @@ def token_set?
       @token && !@token.expired?
     end
 
-    def _metadata_disabled?
+    def ec2_metadata_disabled?
```

Comment:
If we're committing to the chain approach, we can remove this here. Otherwise, revert the chain changes and I can include it in the follow up. Or, you can leave it, and have it do double work.